doc/content/reference/adr/_index.md

```diff
@@ -123,3 +121,43 @@ The `NStep` value would be:
 NStep=int(SNR<sub>margin</sub>/2.5) = int (-5.5/2.5) = -1
 
 In this case, according to the previously mentioned [diagram](https://www.thethingsnetwork.org/forum/uploads/default/original/2X/7/7480e044aa93a54a910dab8ef0adfb5f515d14a1.pdf), if Tx power is lower than its maximum, the Network Server should increase it once by 3dB, while the data rate should not be changed.
+
+### Configure Channel Steering Mode
+
+The channel steering modes are only applicable to `US915` and `AU915` bands. End devices in these bands can join {{% tts %}} network via **LoRa wide channel** mode, i.e. a 500kHz bandwidth channel. This behavior is default, and it can be enabled as follows:
+
+```bash
+ttn-lw-cli end-devices set --application-id <app-id> --device-id <dev-id> --mac-settings.adr.mode.dynamic --mac-settings.adr.mode.dynamic.channel-steering.disabled
+```
+
+The end device can also be steered towards the 8 125kHz channels, instead of using a singular 500kHz channel like mentioned above. This mode is called **LoRa narrow** and it can be enabled with:
```

Comment:
`towards the 125kHz channels, which usually are 8 per frequency sub band (FSB), instead of ...`